kubernetes/chart/zulip/Chart.yaml

```diff
@@ -15,26 +15,26 @@ version: 0.9.20
 appVersion: "9.2-0"
 dependencies:
   - name: memcached
-    repository: https://charts.bitnami.com/bitnami
+    repository: oci://registry-1.docker.io/bitnamicharts
     tags:
       - memcached
-    version: 6.0.16
+    version: 7.4.16
   - name: rabbitmq
-    repository: https://charts.bitnami.com/bitnami
+    repository: oci://registry-1.docker.io/bitnamicharts
     tags:
       - rabbitmq
-    version: 8.32.0
+    version: 14.7.0
   - name: redis
-    repository: https://charts.bitnami.com/bitnami
+    repository: oci://registry-1.docker.io/bitnamicharts
     tags:
       - redis
-    version: 16.8.7
+    version: 20.1.4
   - name: postgresql
-    repository: https://charts.bitnami.com/bitnami
+    repository: oci://registry-1.docker.io/bitnamicharts
     tags:
       - postgresql
     # Note: values.yaml overwrites posgresql image to zulip/zulip-postgresql:14
-    version: 11.1.22
+    version: 15.5.32
```

Comment:
I'm not sure if this change (upgrading across several major PostgreSQL versions) means that folks using the Helm chart may lose their data, or require a manual dump-and-reload process.  I assume you tested across this change, and it Just Worked to upgrade and preserve the backing data in the PostgreSQL server image?